src/Tizen.NUI.Extension/Markup/ViewExtensions.cs

```diff
@@ -283,5 +424,361 @@ public static T BoxShadow<T>(this T view, UIShadow shadow) where T : View
+        /// </summary>
+        /// <typeparam name="T">The type of the view.</typeparam>
+        /// <param name="view">The extension target.</param>
+        /// <param name="visiblity">The visibility value.</param>
+        /// <returns>The view itself.</returns>
+        public static T Visibility<T>(this T view, bool visiblity) where T : View
+        {
+            if (view.Visibility != visiblity)
+            {
+                if (visiblity)
+                {
+                    view.Show();
+                }
+                else
+                {
+                    view.Hide();
+                }
+            }
+            return view;
+        }
+
+        /// <summary>
+        /// Sets the status of whether the view should emit touch or hover signals.
+        /// If a View is made insensitive, then the View and its children are not hittable.
+        /// </summary>
+        /// <typeparam name="T">The type of the view.</typeparam>
+        /// <param name="view">The extension target.</param>
+        /// <param name="sensitive">The sensitive value.</param>
+        /// <returns>The view itself.</returns>
+        public static T Sensitive<T>(this T view, bool sensitive) where T : View
+        {
+            view.Sensitive = sensitive;
+            return view;
+        }
+
+        /// <summary>
+        /// Sets the status of whether the view should be enabled user interactions.
+        /// If a View is made disabled, then user interactions including touch, focus, and actiavation is disabled.
+        /// </summary>
+        /// <typeparam name="T">The type of the view.</typeparam>
+        /// <param name="view">The extension target.</param>
+        /// <param name="enable">The enable value.</param>
+        /// <returns>The view itself.</returns>
+        public static T IsEnabled<T>(this T view, bool enable) where T : View
+        {
+            view.IsEnabled = enable;
+            return view;
+        }
+
+        /// <summary>
+        /// Sets the clipping behavior (mode) of it's children.
+        /// </summary>
+        /// <typeparam name="T">The type of the view.</typeparam>
+        /// <param name="view">The extension target.</param>
+        /// <param name="type">The clipping mode type value.</param>
+        /// <returns>The view itself.</returns>
+        public static T ClippingMode<T>(this T view, ClippingModeType type) where T : View
+        {
+            view.ClippingMode = type;
+            return view;
+        }
+
+        /// <summary>
+        /// Set the layout on this View. Replaces any existing Layout.
+        /// </summary>
+        /// <typeparam name="T">The type of the view.</typeparam>
+        /// <param name="view">The extension target.</param>
+        /// <param name="layout">The layout for the view.</param>
+        /// <returns>The view itself.</returns>
+        public static T Layout<T>(this T view, LayoutItem layout) where T : View
```

Comment:
In addition, It will be great to have extension methods such as,
```C#
view.Size(100, 100)
  .BackgroundColor(UIColor.Black)
  .HorizontalLayout(CellPadding: 10, horizontalAlignment: HorizontalAlignment.Center);
```